include/tests_ssh

```diff
@@ -97,6 +97,9 @@
                 IgnoreRhosts:YES,,NO\
                 UseDNS:YES,,NO\
                 X11Forwarding:NO,,YES\
+                FingerprintHash:SHA256,MD5,\
+                TCPKeepAlive:YES,,NO\
+                Compression:DELAYED,YES,NO\
```

Comment:
Re Compression:

Hmm... Compression is so often a vector for a vulnerability (e.g. BREACH, CRIME, Logjam).  I definitely wouldn't enable it before authentication, and might not enable it at all.  I'd be more inclined to put YES as the "wrong" option.

I don't know of any current vulnerabilities of SSH compression, but that doesn't mean they're not there... and the compression/decompression stack seems to be a good place to look.

Thoughts?